tests/dummy/app/templates/application.hbs

```diff
@@ -1,6 +1,17 @@
 <h2>frost-css-core</h2>
 
-<h3>typography</h3>
+<h3>flex</h3>
```

Comment:
Can we separate the demo out into a demo route per type (e.g. demo/typography/template.hbs) and do links in the application.hbs?  That would make the port into the docs easier

@laflower @EWhite613 comments?